### DISCUSSION TITLE
Ice 0.11.2

### DISCUSSION THREAD

**Discussion by jordanbaird** [jordanbaird/Ice#355] [Announcements]
## What's New
* Bug fixes and UI adjustments.
* Ice's menu bar items now treat Control+clicks as right clicks.

<hr /><em>This discussion was created from the release <a href='https://github.com/jordanbaird/Ice/releases/tag/0.11.2'>Ice 0.11.2</a>.</em>

**Comment by breschnei** [comment]
CGWindowListCreateImage (in your Bridging.swift file) is obsolete in Sequoia. ScreenCaptureKit is the new standard. JFYI.

**Reply by jordanbaird** [reply]
I'm aware of that. In #170, I talk about why we can't use it.
